examples/stereo_agent.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+r"""
+This is a demonstration of how to create an agent with
+two cameras in a stereo pair
+
+
+This can be done by giving the agent two sensors (be it RGB, depth, or semantic)
+with different positions.
+
+Note that the cameras must have different uuid's
+"""
+
+import random
+
+import numpy as np
+
+import habitat_sim
+
+cv2 = None
+
+
+def _render(sim, display, depth=False):
+    for _ in range(100):
+        obs = sim.step("turn_right")
+        stereo_pair = np.concatenate([obs["left_sensor"], obs["right_sensor"]], axis=1)
+
+        if depth:
+            stereo_pair = np.clip(stereo_pair, 0, 10)
+            stereo_pair /= 10.0
+
+        if len(stereo_pair.shape) > 2:
+            stereo_pair = stereo_pair[..., 0:3][..., ::-1]
```

Comment:
As I understand `_render ` isn't considered part of example, but maybe still need more comments or const names for examples to make it clear for new people what's happening here:
`10.0` – threshold const and normalization function/variable name
`[..., 0:3][..., ::-1]` – need explanation